docs/README.md

```diff
@@ -178,25 +178,25 @@
     * [Health Check Library](extensions/healthcheck-library.md)
   * [CA Rotation in Extensions](extensions/ca-rotation.md)
   * Blob storage providers
-    * [`BackupBucket` resource](extensions/backupbucket.md)
-    * [`BackupEntry` resource](extensions/backupentry.md)
+    * [`BackupBucket` resource](extensions/resources/backupbucket.md)
+    * [`BackupEntry` resource](extensions/resources/backupentry.md)
   * DNS providers
     * [`DNSRecord` resources](extensions/dnsrecord.md)
   * IaaS/Cloud providers
     * [Control plane customization webhooks](extensions/controlplane-webhooks.md)
-    * [`Bastion` resource](extensions/bastion.md)
-    * [`ControlPlane` resource](extensions/controlplane.md)
-    * [`ControlPlane` exposure resource](extensions/controlplane-exposure.md)
-    * [`Infrastructure` resource](extensions/infrastructure.md)
-    * [`Worker` resource](extensions/worker.md)
+    * [`Bastion` resource](extensions/resources/bastion.md)
+    * [`ControlPlane` resource](extensions/resources/controlplane.md)
+    * [`ControlPlane` exposure resource](extensions/resources/controlplane-exposure.md)
+    * [`Infrastructure` resource](extensions/resources/infrastructure.md)
+    * [`Worker` resource](extensions/resources/worker.md)
   * Network plugin providers
-    * [`Network` resource](extensions/network.md)
+    * [`Network` resource](extensions/resources/network.md)
   * Operating systems
-    * [`OperatingSystemConfig` resource](extensions/operatingsystemconfig.md)
+    * [`OperatingSystemConfig` resource](extensions/resources/operatingsystemconfig.md)
```

Comment:
Broken link in https://github.com/gardener/gardener/blob/6ba5765c27a1cb90e0fcb11764af7cfe6cc013cd/pkg/component/extensions/operatingsystemconfig/nodeinit/nodeinit.go#L74